packages/x-components/src/x-modules/search-box/components/search-input.vue

```diff
@@ -290,15 +311,21 @@ _Type any term in the input field to try it out!_
 
 ### Play with props
 
-In this example, the search input has been limited to accept a maximum of 5 characters, including
-spaces, it won't take the focus when it is rendered, and it will emit the `UserAcceptedAQuery` event
-after 1000 milliseconds without typing.
+In this example, the search input been limited to accept a maximum of 5 characters (including
+spaces), it won't take the focus when it is rendered, it will emit the `UserAcceptedAQuery` event
+after 1000 milliseconds without typing, and it will prevent users from entering numbers;
 
 _Type a term with more than 5 characters to try it out!_
 
 ```vue live
 <template>
-  <SearchInput :maxLength="5" :autofocus="false" :instant="true" :instantDebounceInMs="1000" />
+  <SearchInput
+    :maxLength="5"
+    :autofocus="false"
+    :instant="true"
+    :instantDebounceInMs="1000"
+    :blacklistedCharacters="0123456789"
```

Comment:
This could not be a number
```suggestion
    blacklistedCharacters="0123456789"
```